server/routes/referrals/referralsController.ts

```diff
@@ -59,6 +59,7 @@ export default class ReferralsController {
     let serviceUser: DeliusServiceUser | null = null
 
     const crn = req.body['service-user-crn']
+    const hardcodedInterventionId = '98a42c61-c30f-4beb-8062-04033c376e2d'
```

Comment:
sorry that this is the second time i've commented a similar thing but

i've added a sample intervention into dev with id `3165ded2-e754-49d5-9c93-e8e063381c56`. so that's the one to use when making the real call. 

for the contract tests and running localy, the existing `98a42c61...` is the one.

i wanted them to be different to ensure there is a clear seperation of the data between local/dev. but it means this hardcoded value is gonne be wrong either for local or dev. i think it should be correct in dev. then we can always change it ourselves when running locally. 

what do you think?

Reply:
hmm, I'd rather we used the same for dev and contract tests, so we can just get things up and running without remembering we'll need to change things locally, as I can imagine that becoming a bit of a pain. When you say you want there to be a separation of data between local and dev, could you see it becoming a problem if we were to re-use it across both?

Reply:
i think this issue only arises from the fact it's hard coded. in general the data will be totally seperate (apart from things that are common across all environments like service category ids). since it's an usual case i'm happy to change the id of the intervention in dev to match the local one.

Reply:
done